packages/chip-input/src/ux-choice-container-attribute.ts

```diff
@@ -0,0 +1,105 @@
+import { UxChoiceAttribute } from './ux-choice-attribute';
+import { customAttribute, bindingMode, bindable, inject } from 'aurelia-framework';
+
+@customAttribute('ux-choice-container', bindingMode.twoWay)
```

Comment:
is `bindingMode.twoWay` added because it wouldn't work otherwise?

Reply:
no, I forgot to remove it after setting the bindable default mode in the @bindable declaration. thanks for the FB